src/components/components/ebay-tooltip-base/index.js

```diff
@@ -30,10 +29,10 @@ function onRender() {
     const isTooltip = this.state.type === 'tooltip';
 
     if (this.host) {
-        this.expander = new Expander(this.el, {
+        this.expander = new Expander(this.el.querySelector(`.${this.state.type}`), {
```

Comment:
should probably use `this.el.getElementsByClassName(this.state.type)[0]` which will be much faster and maybe a bit more clear in this case?